examples/basic-examples/test/e2e/integration/custom/mapping/step_definition.js

```diff
@@ -0,0 +1,15 @@
+/* global cy */
+
+import { Then } from 'cypress-cucumber-preprocessor/steps'
+
+Then(`the example filter is applied`, () => {
+  cy.get('.rpl-hero-banner__title span').should('contain', '[demo]')
```

Comment:
Why not just use the hero banner assertions that already exist in ripple-test-utils?
https://github.com/dpc-sdp/ripple/blob/fe9a82a8d63d4e5c5773775fc2c5fe6c830b3710/packages/ripple-test-tools/step_definitions/common/components.js#L69

https://github.com/dpc-sdp/ripple/blob/fe9a82a8d63d4e5c5773775fc2c5fe6c830b3710/packages/ripple-test-tools/step_definitions/common/components.js#L73

Reply:
As discussed, here I am testing "contain a demo label" not should be match the exact title, so using `the hero banner title should be {string}` sounds not suit for this case.